docker-images/drupal-php/7.1/apache/Dockerfile

```diff
@@ -72,6 +72,9 @@ RUN cd /usr/local \
     && ln -s /usr/local/composer.phar /usr/local/bin/composer \
     && echo 'PATH="$HOME/.composer/vendor/bin:$PATH"' >> $HOME/.bashrc
 
+# Install hirak/prestissimo for improve composer performance
+RUN composer global require hirak/prestissimo
```

Comment:
I think like for the drupal-php-dev image, it would be good to run `composer clear-cache` to lower the image size.

```
# Install Coder and configure Code sniffer.
RUN composer global require drupal/coder:8.2.* \
    && composer global require dealerdirect/phpcodesniffer-composer-installer \
    && composer clear-cache \
    && echo 'alias drupalcs="phpcs --standard=Drupal --extensions='php,module,inc,install,test,profile,theme,css,info,txt,md'"' >> $HOME/.bashrc \
    && echo 'alias drupalcsp="phpcs --standard=DrupalPractice --extensions='php,module,inc,install,test,profile,theme,css,info,txt,md'"' >> $HOME/.bashrc \
    && echo 'alias drupalcbf="phpcbf --standard=Drupal --extensions='php,module,inc,install,test,profile,theme,css,info,txt,md'"' >> $HOME/.bashrc
```